Documentation/cmdref/cilium_bpf.md

```diff
@@ -1,38 +0,0 @@
-<!-- This file was autogenerated via cilium cmdref, do not edit manually-->
```

Comment:
Wait, no, this is probably why the netlify build fails (sorry @qmonnet for the noise!)

Why is this file being deleted in this PR?

Reply:
ah, to resolve the conflicts, I rebased on master and `git rm Documentation/cmdref/cilium_bpf.md`, then `make -C Documentation update-cmdref`, after that, when I saw  `Documentation/cmdref/cilium_bpf.md` get created, I pushed the PR, I should have `git add Documentation/cmdref/cilium_bpf.md; git commit; git push...;" like I usually do

Reply:
it looks I also need to git add Documentation/cmdref/cilium_bpf_vtep.md, Documentation/cmdref/cilium_bpf_vtep_list.md that are created by `make -C Documentation update-cmdref`, I don't remember if  I did this previously for this PR, but now it seems to be the correct way to do it.